cmd/cri-containerd/options/options.go

```diff
@@ -119,13 +134,13 @@ func (c *CRIContainerdOptions) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&c.CgroupPath, "cgroup-path",
 		"", "The cgroup that cri-containerd is part of. By default cri-containerd is not placed in a cgroup.")
 	fs.BoolVar(&c.EnableSelinux, "enable-selinux",
-		false, "Enable selinux support.")
+		false, "Enable selinux support. (default false)")
```

Comment:
Actually, `cri-containerd --help` will show default, we don't need to add this ourselves. I'm with fix this in another PR.

Reply:
Actually, pflag only print default value if the original string doesn't contain the word `default`, which is a bit not smart. :)

So we do need to add some default value description ourselves if the original description contains the word `default`.

Will send a PR to clean this up a little bit.

Reply:
It looked to me that false wasn't printing so I added the false ones that I felts needed to be output.